Assets/Scripts/Lever.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class Lever : MonoBehaviour {
+    public UnityEvent leftOption;
+    public UnityEvent rightOption;
+    private bool left;
+    private Animator animator;
+    private bool inRange;
```

Comment:
These should have default values.

Reply:
Should I declare the default values in these lines instead of at Start()?

Reply:
This default values are used in Unity editor. If you click on Reset component, the component resets to these values. So, it's better to set the default values here rather than in `Start()`.

I think if you write it in `Start()`, it may overwrite editor-configured value, but I am not 100% sure about this one.